commit c0104fc33ccf7e843efd37cc033e4c3849a37844
Author: Christopher Faulet <cfaulet@haproxy.com>
Date:   Sat Sep 28 10:37:31 2019 +0200

    BUG/MINOR: stats: Add a missing break in a switch statement
    
    A break is missing in the switch statement in the function
    stats_emit_json_data_field(). This bug was introduced in the commit 88a0db28a
    ("MINOR: stats: Add the support of float fields in stats").
    
    This patch fixes the issue #302 and #303. It must be backported to 2.0.
    
    (cherry picked from commit 52c91bb72ccf20e75f2f79da6ebe345cb2a768f4)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/src/stats.c b/src/stats.c
index 5819f459..9ec5066f 100644
--- a/src/stats.c
+++ b/src/stats.c
@@ -399,6 +399,7 @@ int stats_emit_json_data_field(struct buffer *out, const struct field *f)
 		       type = "\"u64\"";
 		       snprintf(buf, sizeof(buf), "%llu",
 				(unsigned long long) f->u.u64);
+		       break;
 	case FF_FLT:   type = "\"flt\"";
 		       snprintf(buf, sizeof(buf), "%f", f->u.flt);
 		       break;
